### DISCUSSION TITLE
Does it sync your movie collections?

### DISCUSSION THREAD

**Discussion by FloydKulis** [Taxel/PlexTraktSync#99] [Q&A] [answered]
Does it sync your movie collections? TV series adds to me, but films unfortunately not.

**Comment by simonc56** [comment]
It currently doesn't work with the new Plex Movie agent, it works only with the old one listed as Plex Movie (Legacy).
See issue #56 and PR #83.

**Reply by FloydKulis** [reply]
So I understand that in Plex I have to turn on Plex Movie (Legacy) and filter the entire library again?

**Comment by glensc** [comment, marked as answer]
Version 0.6.42 required then both old and new agents work
